week-2/F-logical-operators/exercise2.js

```diff
@@ -5,7 +5,34 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number) {
+  if (number < 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
```

Comment:
You can just write `return number < 0` for this function. Similarly for the other functions below.